packages/riipen-ui/src/components/Table.jsx

```diff
@@ -0,0 +1,241 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+
+import TableColumn from "./TableColumn";
+import TableHeader from "./TableHeader";
+import TableContainer from "./TableContainer";
+import TableRow from "./TableRow";
+import TableBody from "./TableBody";
+import TableCell from "./TableCell";
+import TableHeaderCell from "./TableHeaderCell";
+
+class Table extends React.Component {
+  static propTypes = {
+    /**
+     * Displays the label instead of the header
+     */
+    allowSearch: PropTypes.bool,
```

Comment:
This is very confusing to me. We have a prop that is called `allowSearch` but it's dealing with labels? Can we rename or update the docs to be slightly more clear as to what is happening?

Reply:
yeah, maybe something like useLabels or renderHeader. We could also say that if you want to disable search functionality it is the job of the website to pass in the component. (Similar to the way the members table does it now) I just thought it would be nice to have it in the UI kit with a simple toggle

Reply:
I think search should actually be up to the using application since it may work differently for different sites.